src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -49,6 +52,75 @@ public static Encoding OutputEncoding
             get { return GetConsoleEncoding(); }
         }
 
+        private static readonly object InternalSyncObject = new object();
```

Comment:
Yeah, I was just referring to the naming.  That said, can we just use Console.EnsureInitialized instead?